TwitchClient/Channel.swift

```diff
@@ -1,27 +0,0 @@
-//
-//  Channel.swift
-//  TwitchClient
-//
-//  Created by Ahmed kotb on 4/8/17.
-//  Copyright © 2017 Ahmed Kotb. All rights reserved.
-//
-
-import UIKit
-
-class Channel: NSObject {
-    
-    var status : String?
-    var language : String?
-    var displayName : String?
-    var logo : String?
-    
-    func populateChannelWithData(DataDictionary : NSDictionary) -> Channel
```

Comment:
Also why are you using `NSDictionary` not just `Dictionary`? I feel you're consistently trying to write Objective-C code but in Swift, you're losing valuable Swift features and optimizations. Even the naming of the method is wrote like you would in Objective-C